core/domain/question_domain.py

```diff
@@ -415,7 +402,8 @@ def __init__(
         """
         self.id = question_id
         self.creator_id = creator_id
-        self.question_content = html_cleaner.clean(question_content)
+        self.question_content = html_validation_service.convert_to_ckeditor(
```

Comment:
Yes, we need this in cases where user copy pastes some stuff which can be converted to a valid ckeditor html format but the validation fails since it is not initially in ckeditor format. So, we should initially have the cleaned up html here to avoid this failure. 

One of the examples (for subtitled html) is when we add text `Congratulations! Exploration is finished` for end exploration by default, the validation will fail but if we convert this to ckeditor format by wrapping in `p` tags, it becomes a valid html.

Reply:
Ah ok, please add comments to explain, at the points where you use it. This is a bit subtle and it's good to give devs the reminder.

(In general I think you might want to do this automatically for stuff that's not obvious and that you get questions about during reviews.)

Reply:
Done. Yes, got it. Will add comments from now on for stuff that's not obvious.